caas/constants.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package caas
+
+const (
+	// CharmContainerName is the name of the charm container in a charm application pod.
+	CharmContainerName = "charm"
```

Comment:
We already have `ApplicationCharmContainer = "charm"`

Reply:
resolved with thanks ian!